resources/dcs_first_boot.sh

```diff
@@ -1,20 +1,90 @@
 #!/bin/bash
```

Comment:
If this script is not used, it should be completely deleted...

Reply:
I do not understand (this also concerns your main commentary) first_boot is used. It was moved to specific method, that's why you might think it is unused, but it is here: https://github.com/airvolute/dcs-deploy/blob/custom_rootfs_and_batch/dcs_deploy.py#L688-L746

First boot is very important, there are plenty of things happening, such as networking setup, usb HW setup, etc etc.